### PR TITLE
Fix pre-commit typing python version

### PR DESCRIPTION
**Pull request by LecrisUT**
Just figured out, when `language_version` is not specified, the `python_version` in `additional_dependencies` is ignored, although the configuration in `pyproject.toml` is still respected.

@happz, you might want to pull this one early for the typing fixes

### CODE REVIEW

File: tests/unit/test_utils.py [hunk 1]
```diff
@@ -1028,9 +1028,9 @@ def test_import_member(root_logger):
 
 def test_import_member_no_such_module(root_logger):
     with pytest.raises(
-            SystemExit,
-            match=rf"Failed to import the 'tmt\.steps\.nope_does_not_exist'"
-                  rf" module from '{Path.cwd()}'."):
+        SystemExit,
+        match=rf"Failed to import the 'tmt\.steps\.nope_does_not_exist'"
+            rf" module from '{Path.cwd()}'."):
```
Comment (happz) [inline tests/unit/test_utils.py:1033]: Seems like an unrelated change. A valid one, but unrelated, or is it caused by a linter reformating the file?

Comment (happz) [reply tests/unit/test_utils.py:1033]: Aaaha, got it. I didn't suspect autopep8 to cause this, but it's true some linters do behave like this.

**Review by happz — APPROVED** [review]

**Review by therazix — APPROVED** [review]
